simulator/vehicles/iris-ap/iris.launch.xml

```diff
@@ -5,14 +5,15 @@
     <arg name="sim_only" default="false"/>
 
     <include unless="$(var spawn_only)" file="/ros_ws/launch/gzweb.launch.xml" />
-    
+
+    <!-- Launch spawn entity after deplay to ensure sdf has been generated -->
```

Comment:
Is this still valid? SDF is now generated in the as part of the entrypoint scripts.

Reply:
The comment just needed updating, still need the launch to spawn entity!